activejob/test/models/custom_type.rb

```diff
@@ -0,0 +1,16 @@
+class CustomType
+  # Enable custom serialization for this class
+  include ActiveJob::Serialization
+  # Some attributes to serialize
+  attr_reader :foo, :bar
+
+  def initialize(v1, v2)
+    @foo = v1
+    @bar = v2
+  end
+
+  # equality method used in tests
+  def ==(obj)
```

Comment:
I'd inline it, but define the constant outside of the test case (still inside the test class though)